adapters/repos/db/shard_test.go

```diff
@@ -231,7 +231,7 @@ func TestShard_InvalidVectorBatches(t *testing.T) {
 
 func TestShard_DebugResetVectorIndex(t *testing.T) {
 	t.Setenv("ASYNC_INDEXING", "true")
-	t.Setenv("ASYNC_STALE_TIMEOUT", "200ms")
```

Comment:
same here, should we keep this env variable for backward compatibility reasons? or it's ok to replace it with a new